modules/apps/object/object-service/service.xml

```diff
@@ -446,6 +446,7 @@
 		<column name="name" type="String" />
 		<column name="reverse" type="boolean" />
 		<column name="type" type="String" />
+		<column name="parameterObjectFieldId" type="long" />
```

Comment:
Shouldn't the `parameterObjectFieldId` be right before the `reverse` column?

Reply:
As discussed offline, I'll follow previous precedent.